src/views/tutorial-view/components/next-previous/index.tsx

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-import DirectionalLinkBox from './components/directional-link-box'
+import DirectionalLinkBox from '../../../../components/directional-link-box'
```

Comment:
Can we add an alias for components in the tsconfig?

Reply:
Let's do that in another PR.